src/main/java/uk/gov/hmcts/reform/lrdapi/util/ValidationUtils.java

```diff
@@ -148,14 +145,9 @@ public static boolean isListContainsTextIgnoreCase(List<String> idList, String s
         return idList.stream().anyMatch(searchText::equalsIgnoreCase);
     }
 
-    public static void checkRegionDescriptionIsValid(String region) {
-
-        if (isBlank(region)) {
-            throw new InvalidRequestException("No Region Description provided");
-        }
-
-        if (!Pattern.compile(REGION_NAME_REGEX).matcher(region).matches()) {
-            throw new InvalidRequestException(EXCEPTION_MSG_REGION_SPCL_CHAR);
+    public static void checkStringContainsMoreThanOneConsecutiveComma(String param) {
+        if (param.contains(",,")) {
```

Comment:
Please use `StringUtils.startWith()` and `StringUtils.endsWith()` methods to test this. I have added a utility method in the PR to exactly perform the same action. Refer to the method - `checkIfValidCsvIdentifiersAndReturnList()` in the file - https://github.com/hmcts/rd-location-ref-api/pull/274/files#diff-2c65a212b41b3ecc8d1d8ceb4561baf1e6d35711db7e2b1f76d435ba0d464f8b

Reply:
Addressed here: https://github.com/hmcts/rd-location-ref-api/pull/273/commits/c4e815b456384b88ed02edb15e4e10d4c0c521ca